content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -32,6 +35,15 @@ To learn how Istio handles tracing, visit this task's [overview](../overview/).
 1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage` one or more times
     to generate trace information.
 
+    The number of requests you need to send in order to see trace data depends on the sampling rate set at install.
+    By default, the sampling rate is set to 1%, meaning that you need to send at least 100 requests.
```

Comment:
requets. -> requests before the first trace will be visible.

Reply:
Done.